.github/workflows/ci.yml

```diff
@@ -274,18 +276,25 @@ jobs:
               url="https://boringssl.googlesource.com/boringssl/+archive/${{ matrix.library.version }}.tar.gz"
               tar_flags=""
               ;;
+            "aws-lc")
+              url="https://github.com/aws/aws-lc/archive/refs/tags/${{ matrix.library.version }}.tar.gz"
+              tar_flags="--strip-components=1"
+              ;;
             esac
 
             case "${{ matrix.target}}" in
             "x86_64-unknown-linux-gnu")
+              CPU=x86_64
               OS_COMPILER=linux-x86_64
               OS_FLAGS=""
               ;;
             "i686-unknown-linux-gnu")
+              CPU=i686
               OS_COMPILER=linux-elf
               OS_FLAGS="-m32 -msse2"
               ;;
             "arm-unknown-linux-gnueabihf")
+              CPU=armv4
```

Comment:
According to the notes [here](https://doc.rust-lang.org/beta/rustc/platform-support.html#tier-2-with-host-tools), this target is `Armv6 Linux, hardfloat (kernel 3.2, glibc 2.17)`. 

So, "armv6"?

Reply:
This isn't the Rust target definition in this case, this is the compiler being used to precompile the crypto library, so either aws-lc, openssl, or boringssl. The line immediately below this one indicates its an armv4 compiler being used?